python/shotgun_globals/cached_schema.py

```diff
@@ -507,4 +534,43 @@ def get_status_color(cls, status_code):
         return status_color
 
 
+    @classmethod
+    def field_is_editable(cls, sg_entity_type, field_name):
+        """
+        Returns a boolean identifying the editability of the entity's field.
+
+        :param str sg_entity_type: the entity type
+        :param str field_name: the field name to check editibility
+
+        :returns: ``True`` if the field is ediable, ``False`` otherwise.
+        :rtype: :obj:`bool`
+        """
+        self = cls.__get_instance()
+        self._check_schema_refresh(sg_entity_type, field_name)
+
+        if sg_entity_type in self._type_schema and field_name in self._field_schema[sg_entity_type]:
+            data = self._field_schema[sg_entity_type][field_name]
+            return data.get("editable", {}).get("value", False)
```

Comment:
Is there a case where editable isn't there?
I'm just wondering what it is, since the default to False here could have implications.

I think examples would help clarify if this should be False, True, or an error condition.